game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java

```diff
@@ -701,11 +706,7 @@ public void paint(final Graphics g) {
     }
     for (final Collection<Unit> value : highlightedUnits) {
       for (final UnitCategory category : UnitSeparator.categorize(value)) {
-        final List<Unit> territoryUnitsOfSameCategory = category.getUnits();
-        if (territoryUnitsOfSameCategory.isEmpty()) {
-          continue;
-        }
-        final Rectangle r = tileManager.getUnitRect(territoryUnitsOfSameCategory, gameData);
```

Comment:
Good change (I assume this is the only usage of the getUnitRect method)

Reply:
Yep!